Source/Particles/Resampling/VelocityCoincidenceThinning.H

```diff
@@ -160,27 +169,35 @@ public:
                 const int ii = static_cast<int>((ux[indices[i]] - ux_min) / dux);
                 const int jj = static_cast<int>((uy[indices[i]] - uy_min) / duy);
                 const int kk = static_cast<int>((uz[indices[i]] - uz_min) / duz);
+                amrex::ParticleReal w_i = w[indices[i]];
```

Comment:
```suggestion
                const amrex::ParticleReal w_i = w[indices[i]];
```
This should fix the clang-tidy CI test